### PR TITLE
CI setup docs

### PR DESCRIPTION
**Pull request by maelle**
This PR makes the recommended method easier to find, and adds a link to a package using it.

One thing I'd also like to add, but maybe by a separate PR, is a TOC in order to make it easier to navigate to the available linters and project setup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,16 +179,38 @@ lintr::lint_package() %>%
...
+* either using `lintr::lint_package` as a separate step in your build process (cf [subsection below](#nonfailinglints), which we recommend,
+
+* or setting an unit test (cf [subsection below](#testthat)), which might take a long time to run and hinder development.
+
+In both cases the [lintr-bot](https://github.com/lintr-bot) will add comments
+to the commit or pull request with the lints found and they will also be
+printed on Travis-CI or Wercker.  If you want to disable the commenting you can
+set the environment variable `LINTR_COMMENT_BOT=false`.
+
+### Non-failing Lints <a href="#nonfailinglints" name="nonfailinglints"></a>
```
Comment (jimhester) [inline README.md]: Github automatically generates links for headers, so you don't need to make a new link yourself, the links above should be to `#non-failing-lints` and ``#testthat` below.

File: README.md [hunk 2]
```diff
@@ -179,16 +179,38 @@ lintr::lint_package() %>%
...
-### Testthat ###
+There are two strategies for getting `lintr` results: 
+
+* either using `lintr::lint_package` as a separate step in your build process (cf [subsection below](#nonfailinglints), which we recommend,
+
+* or setting an unit test (cf [subsection below](#testthat)), which might take a long time to run and hinder development.
+
+In both cases the [lintr-bot](https://github.com/lintr-bot) will add comments
+to the commit or pull request with the lints found and they will also be
+printed on Travis-CI or Wercker.  If you want to disable the commenting you can
```
Comment (jimhester) [inline README.md]: I think we can remove the reference to Wercker here (I realize it was copied from the previous state).

**Comment by maelle** [comment]
Changes made!

Are you interested in another PR adding a TOC?

**Comment by jimhester** [comment]
> Are you interested in another PR adding a TOC?

Sure!

**Comment by jimhester** [comment]
Thanks!

**Pull request merged**